core/src/main/java/org/alfresco/httpclient/HttpClient4Factory.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * #%L
+ * Alfresco Records Management Module
```

Comment:
Are the copyright modules accurate? It doesn't seem to fit with Records Management here